test/headless/specs/utilities.js

```diff
@@ -19,8 +21,9 @@ testModule('elementSatisfiesRatio', class extends TestClass {
         window.STATE.satisfied = true;
       })
     })
-    .wait(30)
+    .wait(constants.time.RAF_THRESHOLD * 5)
```

Comment:
Ok. We can get this merged and then hopefully we won't have to do this once we move to another headless driver